app/controllers/concerns/error_extender.rb

```diff
@@ -40,4 +40,10 @@ def render_jsonapi_not_authorized
     error = { status: '403', title: Rack::Utils::HTTP_STATUS_CODES[403] }
     render jsonapi_errors: [error], status: :forbidden
   end
+
+  def render_jsonapi_custom_not_found(exception, use_sentry: true)
+    report_to_sentry(exception)
```

Comment:
Is there supposed to be an `if use_sentry` conditional here?

Reply:
oh *facepalm*, yeah.

Reply:
Removing the sentry log on second thought, as I dont think it's an error that we need to document.